src/cascadia/TerminalConnection/AzureConnection-ARM64.cpp

```diff
@@ -0,0 +1,47 @@
+#include "pch.h"
```

Comment:
nit: pretty sure this needs a copyright header

Reply:
Done